client/my-sites/pages/page/index.js

```diff
@@ -374,9 +375,9 @@ class Page extends Component {
 	undoPostStatus = () => this.updatePostStatus( this.props.shadowStatus.undo );
 
 	render() {
-		const { editorUrl, page, shadowStatus, translate, preventEditingInGutenberg } = this.props;
+		const { editorUrl, page, shadowStatus, translate, isPostsPage: latestPostsPage } = this.props;
```

Comment:
The linter was complaining here since `isPostsPage` was already declared in upper scope.